gemini-k8s-cluster

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Convenient script to run gemini on cluster
+#
+# N.B:
+# make sure kubectl has the same version as server
+
+# variables
+declare -r DB_HOST='scylladb'
+declare -r SPARK_MASTER='spark://p-spark-master:7077'
+
+# create pod if not exists
+if ! kubectl get pods | grep gemini > /dev/null; then
+  kubectl create -f ./k8s/gemini-pod.yaml
+fi
+
+while ! kubectl get pods | grep "gemini.*Running" > /dev/null; do
+  sleep 1
```

Comment:
`while` loop will run forever if the pod cannot be created, won't it?

Reply:
yes. because creating can take second, minutes or hours. It depends on many factors. You will have to investigate it manually anyway.